draft-ietf-quic-transport.md

```diff
@@ -2427,6 +2407,32 @@ ACK Block (repeated):
   packets preceding the largest packet number, as determined by the
   preceding Gap.
 
+### Sending ACK Frames
+
+Implementations MUST NOT generate packets that only contain ACK frames in
+response to packets which only contain ACK frames. However, they SHOULD
```

Comment:
I think that we want to MUST here.  Requiring that all received packets be acknowledged is good for the protocol.